tests/numerics/ad/test_operator_functions.py

```diff
@@ -0,0 +1,78 @@
+"""Test collection testing Ad operator functions.
+
+Testing involves their creation, arithmetic overloads and parsing.
+
+"""
+from __future__ import annotations
+
+import pytest
+
+import numpy as np
+import porepy as pp
+
+
+def test_ad_function():
```

Comment:
Basic test for operator functions.
Since the values of various ad.functions is tested separately, this test covers the general usage of `Function`